app/controllers/admin/courses_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Admin
+  class CoursesController < ApplicationController
+    DEFAULT_FIELDS = [:id, :name, :is_hidden, :display_name].freeze
+    before_action { authorize! }
+
+    respond_to :html
+    layout 'assignment_content'
+
+    def index
+      respond_to do |format|
+        format.html
+        format.json { render json: Course.order('id').to_json(only: DEFAULT_FIELDS) }
```

Comment:
One of the recommendations from the Rails style guide is to order by `:created_at` instead of `:id`. (I haven't enabled the corresponding Rubocop check yet since it affects a bunch of places, but for new code we should follow the rule.)